subjects/devops/orchestrator/audit/README.md

```diff
@@ -0,0 +1,232 @@
+#### General
+
+##### Check the Repo content
+
+A `README.md` and `orchestrator.sh` and `Vagrantfile` files and all files used to create and delete and manage the student infrastructure must be submitted in the repo.
+
+```console
+.
+├── Manifests
+│   └── [...]
+├── Scripts
+│   └── [...]
+├── Dockerfiles
+│   └── [...]
+└── Vagrantfile
+```
+
+###### Are all the required files present?
+
+##### Ask the following questions to the group or student
+
+- What are Container orchestration and what are its benefits?
+
+- What is Kubernetes and what is its main role of it?
+
+- What are K3s and what is their main role?
+
+###### Did the student reply correctly to the questions?
+
+##### Check the Student Documentation
+
+The Documentation must exist in the `README.md` file!
+
+###### Did the README.md file contains the all required information about the solution(Prerequisites, Configuration, Setup, Usage, ...)?
+
+##### Check the docker images in the docker hub:
+
+###### Do the used docker images in the YAML manifest uploaded in the student docker hub account?
+
+##### Check the cluster:
+
+By using k3s in Vagrant 2 virtual machines must be created:
+
+1. Master: the master in the k3s cluster.
+
+2. Agent: an agent in the k3s cluster.
+
+`kubectl` must be installed and configured in the learner machine to manage the cluster.
+
+The nodes must be connected and available, try:
+
+```console
+$> kubectl get nodes -A
+NAME                                           STATUS   ROLES    AGE    VERSION
+<master-node>   Ready    <none>   XdXh   vX
+<agent1-node>   Ready    <none>   XdXh   vX
+$>
+```
+
+###### Does the cluster created by a Vagrantfile?
+
+###### Does the cluster contains 2 nodes (master and agent)?
+
+###### Does kubectl installed and configured in the learner's machine
```

Comment:
Does -> Is

Also don't forget to add `?`

Reply:
fixed